README.md

```diff
@@ -18,7 +16,7 @@ Even if you docker save the manually configured container, the steps to reproduc
 The key point to take-away from the sections below is that your application Dockerfile should always follow a pattern similar to:
 
 ```dockerfile
-FROM websphere-liberty:kernel
+FROM websphere-liberty:kernel-java8-openj9-ubi
```

Comment:
needs the `ibmcom/` prefix (same for the other ones)

Reply:
Added `ibmcom/` to the `FROM` statements and I will address the other readme changes regarding the liberty version change in a separate issue.

Reply:
thanks!